Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/LATISS-Combined-Calibrations-Procedure/latiss-combined-calibrations-procedure.rst

```diff
@@ -13,30 +13,33 @@ Latiss Combined Calibrations Generation Procedure
 
 The script currently has the option (via the ``script_mode`` parameter in the configuration options) to:
 
 - take only biases, 
 - take biases and darks, and 
 - take biases, darks, and flats. 
   
-These options are constrained by the fact that the generation or construction of one calibration depends on the existence of the previous one (i.e., to generate a combined dark, a combined bias is necessary, and to generate a flat, a combined dark and a combined bias are necessary). Calibration generation from the images taken can be skipped by setting ``generate_calibrations``. This will speed up the execution time of the script, and subsequent tasks (for example, verification tasks or the PTC construction task) will look for necessary calibrations in their input collections (whose default is the standard calibrations collection: ``LATISS/calib``).
+These options are constrained by the fact that the generation or construction of one calibration depends on the existence of the previous one (i.e., to generate a combined dark, a combined bias is necessary, and to generate a flat, a combined dark and a combined bias are necessary). If ``generate_calibrations`` is ``True``, calibrations will eventually end up in the collection specified by the ``calib_collection`` configuration parameter and, if certified, will have a validity span range from ``certify_calib_begin_date`` to ``certify_calib_end_date``.
 
-If desired, defects can be constructed from darks and flats, and a PTC per detector per amplifier constructed from the flats. Note that the PTC assumes that a sequence of flat pairs has been taken, each pair taken at the same exposure time. In both cases, ``script_mode`` must be set to ``BIAS_DARK_FLAT``.
+Calibration generation from the images taken can also be skipped by setting ``generate_calibrations`` to ``False``. This will speed up the execution time of the script, and subsequent tasks (for example, verification tasks or the PTC construction task) will look for necessary calibrations in their input collections (whose default is the standard calibrations collection: ``LATISS/calib``).
```

Comment:
I think it would help to be specific and list the steps that it skips.

Reply:
Sorry, I'm confused here. The step that will be skipped when ``generate_calibrations``is ``False`` is that step of generating calibrations, which is what the sentence here says.

Reply:
If you don't build the calibrations, when thy do verification tasks still run?